myapp/app/controllers/task_controller.rb

```diff
@@ -42,7 +42,7 @@ def update
   def destroy
     @task = Task.find(params[:id])
     @task.destroy
-    flash[:notice] = "Task deleted."
+    flash[:notice] = t('task.flashes.success.deleted')
     redirect_to task_index_path, status: 303
   end
 
```

Comment:
Method `task_params` does not have the correct indention; after the private keyword, there should be a blank line. Please fix those as well.

Reply:
fixed